internal/routes/all_routes.go

```diff
@@ -21,7 +30,7 @@ func pathPrefix() string {
 
 func apiRouter() http.Handler {
 	r := chi.NewRouter()
-	r.Use(identity.EnforceIdentity)
+	r.Use(identity.EnforceIdentity, redocMiddleware)
```

Comment:
This code inserts a documentation middleware to all API requests. I do not understand why we would need a middleware for data endpoints, this slows down the request processing and adds another function call to the stack which can become crowded very soon.

I suggest to create a path that will be dedicated for OpenAPI, something like:

```go
r.Route("/openapi", func(r chi.Router) {
 r.Use(redocMiddleware)
 r.Get("/spec.json", func(w http.ResponseWriter, r *http.Request) {
  http.ServeFile(w, r, *config.OpenAPIPath)
  })
}
```

This way the middleware will only be used for this particular path. Also identity is not really needed for OpenAPI docs, we want to advertise our API, make it as easy to learn as possible. That is the purpose of this PR and your change is a huge contribution. Let's not "hide" it behind some obscure custom HTTP header for no reason.